content/en/learn.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Learn
+title: Documentation
 sidebar: false
 ---
 
-To learn how to use NumPy, you could begin with the resources listed under the **'Beginners'** section. Once you gain a broad view, you could try the more involved tutorials and resources listed under the **'Advanced'** section.
+<p style="font-size: 140%;">The official NumPy documentation lives at <a href="https://numpy.org/doc">https://numpy.org/doc.</a></p>
 
-This is a curated collection of NumPy related educational resources. Some are very specific to NumPy, while others offer a broader view on numerical computing. It is a continuously evolving list. In case you would like to contribute, please refer to the information at the bottom of this page.
+Below is a curated collection of external resources. To contribute, see [below](#add-to-this-list).
```

Comment:
Much more concise, nice. The one tweak I'd like to make is remove the double use of "below".

Reply:
+1, missed that.